From 635da3490810ae4c190aa53bacfc43aa8bf004c7 Mon Sep 17 00:00:00 2001
From: Sebastien Bacher <seb128@ubuntu.com>
Date: Wed, 2 Dec 2015 13:46:24 +0100
Subject: GtkAppChooserWidget: wrap the label, use a standard icon

Let the label wrap if needed, that's required in some locales, use an
icon which is available in the theme

https://bugzilla.gnome.org/show_bug.cgi?id=758908
---
 gtk/ui/gtkappchooserwidget.ui | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/gtk/ui/gtkappchooserwidget.ui b/gtk/ui/gtkappchooserwidget.ui
index 015bdd6..e290c02 100644
--- a/gtk/ui/gtkappchooserwidget.ui
+++ b/gtk/ui/gtkappchooserwidget.ui
@@ -103,7 +103,7 @@
             <child>
               <object class="GtkImage">
                 <property name="visible">1</property>
-                <property name="icon-name">gnome-software-symbolic</property>
+                <property name="icon-name">application-x-executable-symbolic</property>
                 <property name="pixel-size">48</property>
                 <property name="margin">12</property>
                 <style>
@@ -117,6 +117,7 @@
                 <property name="label" translatable="yes">No applications found.</property>
                 <property name="halign">center</property>
                 <property name="valign">center</property>
+                <property name="wrap">1</property>
                 <attributes>
                   <attribute name="scale" value="1.2"/>
                 </attributes>
-- 
cgit v0.11.2


